src/components/ApplicationMenu/ApplicationMenu.js

```diff
@@ -154,21 +163,31 @@ class ApplicationMenu extends Component<Props> {
     // During onboarding, there is no selected project (because none exists
     // yet). Therefore, we only want to show the 'Project' menu when a project
     // is selected.
-    if (selectedProjectId && devServerTask) {
+    if (selectedProjectId) {
       // The `Project` menu should be inserted right after `Edit`, which will
       // have a different index depending on the platform.
       const editMenuIndex = template.findIndex(menu => menu.id === 'edit');
 
+      // Only include clear console menu item if devServerTask exists
+      let submenu = [];
+
+      if (devServerTask) {
+        submenu.push({
+          label: isMac ? 'Clear Server Logs' : 'Clear server logs',
+          click: () => clearConsole(devServerTask),
+          accelerator: 'CmdOrCtrl+K',
+        });
+      }
+
+      submenu.push({
+        label: isMac ? 'Delete Project' : 'Delete project',
+        click: () => showDeleteProjectPrompt(selectedProject),
+      });
+
       template.splice(editMenuIndex, 0, {
         id: 'project',
         label: isMac ? 'Project' : '&Project',
```

Comment:
While I was testing your changes locally, I found I kept expecting "Create new project" to be within the `Project` menu.

Unrelated to this PR, and non-blocking, but what do you think about us renaming this to `Current Project`? That way it's clear that this menu is meant to deal with the currently-selected project?

Reply:
👍 Yeah I agree, I think it would be more descriptive.